### ISSUE TITLE
Doesn't work in neovim 0.4.4

### ISSUE THREAD

**Issue by Vftdan** [jbyuki/instant.nvim#35]
`vim.g` is `nil` in neovim 0.4.4

`E5108: Error while calling lua chunk for luaeval(): ...vftdan/.config/nvim/plugged/instant.nvim/lua/instant.lua:99: attempt to index field 'g' (a nil value)`

**Comment by alerque** [comment]
The current stable version of NeoVIM is v0.8.3.

I'm not the developer of this plugin, but I would not expect actively developed plugins using modern editor features to be backwards compatible that far. That's a long way back and NeoVIM has done a ton of work that give a lot more power and convenience to plugins.

If you want plugins that work with such an old editor you will probably need to lookup the release date of the editor and then checkout old versions of plugins from the same general time range, maybe up until the release date of the next major version.

**Comment by hatkidchan** [comment]
Plugin claims to require version 0.4.4 at least and it doesn't work on it. Maybe README is outdated?

**Comment by alerque** [comment]
Yes, I would expect the README is outdated and the easiest "fix" for this issue is to figure out what the actual minimum NeoVIM is these days.
